shards/core/runtime.cpp

```diff
@@ -3032,7 +3033,6 @@ void shInit() {
   shInterface.set("createMesh", emscripten::val(reinterpret_cast<uintptr_t>(iface->createMesh)));
   shInterface.set("destroyMesh", emscripten::val(reinterpret_cast<uintptr_t>(iface->destroyMesh)));
   shInterface.set("schedule", emscripten::val(reinterpret_cast<uintptr_t>(iface->schedule)));
-  shInterface.set("unschedule", emscripten::val(reinterpret_cast<uintptr_t>(iface->unschedule)));
```

Comment:
What's the new way to unscheduled now? Or just not possible in the new C API?

Reply:
it's better to stop the wire, always was

Reply:
but I could revert that bit